input/start/index.md

```diff
@@ -2,7 +2,7 @@
 title: Getting started
 ---
 
-# Getting started with Skupper
+# Getting started with Skupper CLI on Kubernetes
```

Comment:
By design, there is just one primary "Getting started" experience on the website, with notes about how to apply it to different flavors of Kubernetes. What you've got here bifurcates things at the top level between "Skupper CLI" on the one hand and "OpenShift" on the other.